frontend/src/components/posts/PostForm.jsx

```diff
@@ -5,38 +5,37 @@ import React, { useState } from 'react';
 import ResponsiveTextArea from '../ResponsiveTextArea';
 
 // Project Files
```

Comment:
Please could you remove these lines? Looks like there are two `// Project files` comments